src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1475,6 +1475,337 @@ def unfuse_lora(self, components: List[str] = ["transformer", "text_encoder", "t
         super().unfuse_lora(components=components)
 
 
+class AuraFlowLoraLoaderMixin(LoraBaseMixin):
+    r"""
+    Load LoRA layers into [`AuraFlowTransformer2DModel`]
+    Specific to [`AuraFlowPipeline`].
+    """
+
+    _lora_loadable_modules = ["transformer"]
+    transformer_name = TRANSFORMER_NAME
+    text_encoder_name = TEXT_ENCODER_NAME
```

Comment:
Do we need the `text_encoder_name` then?

Reply:
Sorry I missed that, has been removed.